audit/projects/kubernetes-public/service-accounts/boskos-janitor-test@kubernetes-public.iam.gserviceaccount.com/iam.json

```diff
@@ -0,0 +1,11 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "serviceAccount:kubernetes-public.svc.id.goog[test-pods/boskos-janitor]"
```

Comment:
Why is it still called "test-pods"

Reply:
See https://github.com/kubernetes/k8s.io/pull/806#discussion_r416215390